### PR TITLE
Set marker command in mission file

### PR DESCRIPTION
**Pull request by Philipp66904**
This PR introduces a new command in the MDF:
The `set_marker` command basically skips the QR code scanning and directly sets a new marker as if it was detected.
This allows for easier and faster adaptions if the qr code scanning fails, as we need to adapt our files in this case.
This way, we can simply replace `detect_marker` with `set_marker`.

Additionally, the unit tests are now complete.

### CODE REVIEW

**Comment by Philipp66904** [comment]
~Please note, that the test coverage is currently not correct.~
~This is the case, because it requires changes in the `common_package` which have not been merged yet.~

Edit: They are merged now

**Comment by Philipp66904** [comment]
This is ready for review @J0hn-Lawrence @J0las

**Review by J0las — APPROVED** [review]
Looks good

**Comment by J0las** [comment]
Do we want to keep this from merging, until Finn has made some progress on the qr package?

**Comment by Philipp66904** [comment]
> Do we want to keep this from merging, until Finn has made some progress on the qr package?

This PR is completely independent of Finn and his changes (this PR just introduces new functionality).
That's why I plan to merge this as soon as possible.

**Comment by J0hn-Lawrence** [comment]
Can you explain the new set marker functionality to me in a few words, @Philipp66904 ? I am a bit confused as to what it is and don't want to waste time trying to find out myself instead of just asking you.

**Comment by Philipp66904** [comment]
> Can you explain the new set marker functionality to me in a few words, @Philipp66904 ? I am a bit confused as to what it is and don't want to waste time trying to find out myself instead of just asking you.

TL;DR: It skips the qr code scanning and just assumes that a specific marker was decoded.

These two have the same result in our mission progress (if the qr code scanner successfully decoded "1"):

```javascript
{
    "type": "detect_marker",
    "data": {
        "timeout_ms": 30000
    }
}
``` 

```javascript
{
    "type": "set_marker",
    "data": {
        "marker_name": "1"
    }
}
```
In both cases, the commands for marker "1" will be executed next.

**Comment by Philipp66904** [comment]
This allows us to remove the qr code scanning in the mdf quickly if we see that it doesn't work at the challenge day.
The LT guys need to fly their mission and probably only have limited batteries available.
If we fail to scan the qr code (e.g. due to bad lighting conditions, ...), we can simply replace a few lines in our mdf and thereby be independent of the qr code scanning (and showcase the successfull scanning capabilities in the lab or at our own flight days instead).
Previously, a lot of changes were needed to adapt the mdf and get the same result.

To sum it up: It provides us with more flexibility and a quicker plan B if something isn't working as expected. The qr code scanning behavior is not impacted by this change and still is plan A.

**Comment by Philipp66904** [comment]
@J0hn-Lawrence

**Review by finn-ehlers — APPROVED** [review]
